ci/steps.yml

```
@@ -244,11 +244,13 @@ steps:
     }
     cd $(WIN_QT_BINDIR)
     $jomurl = 'https://download.qt.io/official_releases/jom/' + $env:JOM
-    $md5sum = $env:MD5 + '  ' + $env:JOM
     Invoke-WebRequest -Uri $jomurl -OutFile $env:JOM
-    Set-Content -Encoding utf8 -Path md5sums.txt -Value $md5sum
-    md5sum -c md5sums.txt
-    unzip $env:JOM
+    if ((Get-FileHash $env:JOM -Algorithm MD5).Hash -eq $env:MD5) {
+      unzip $env:JOM
+    } else {
+      Write-Host "##[error]Invalid MD5 hash for a downloaded $env:JOM ... aborting" 
+      exit 1
+    }
```

No `else`; change "invalid" to "unmatched"; make style consistent.

```suggestion
    if ((Get-FileHash $env:JOM -Algorithm MD5).Hash -ne $env:MD5) {
      Write-Host "##[error]Unmatched MD5 hash for a downloaded $env:JOM ... aborting" 
      exit(1)
    }
    unzip $env:JOM
```

We don't need `##vso[task.logissue type=error]`?

https://github.com/miurahr/aqtinstall/blob/0ffe65700c50c8175a80b804e579df7b4ee99da3/ci/steps.yml#L460-L470

---

Btw, there is a typo: 

https://github.com/miurahr/aqtinstall/blob/0ffe65700c50c8175a80b804e579df7b4ee99da3/ci/steps.yml#L468

```diff
-nmake 
+make

```

.